templates/examples/cryptocurrency-slack-bot/index.js

```diff
@@ -0,0 +1,302 @@
+addEventListener("fetch", (event) => {
+	event.respondWith(slackWebhookHandler(event.request));
+});
+
+// SLACK_TOKEN is used to authenticate requests are from Slack.
+// Keep this value secret.
+let SLACK_TOKEN = "PUTYOURTOKENHERE";
```

Comment:
Uhhhhh. This should be a secret binding. Not a blocker for this PR, obviously (this being copied over from the templates repo), but these could probably do with a security review at some point

Reply:
Should make this a ticket.